### PR TITLE
chore(docs): convert demo divs to iframes

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

**Review by YonatanKra — COMMENTED** [review]
Looks nice. Can merge like it IMO, but the suggestions will make the code more readable IMO.

File: apps/docs/transformers/code-block-demo.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+const path = require('path');
+const ELEVENTY_HTML_CODE_BLOCK_SELECTOR = 'pre > code.language-html';
+
+const CBD_BASE = 'cbd-base';
+const CBD_DEMO = 'cbd-demo';
+const CBD_DETAILS = 'cbd-details';
+const CBD_BUTTON_SHOW = 'cbd-button-show';
+const CBD_CODE_BLOCK = 'cbd-code-block';
+
+const generateCodeBlockDemo = function (pre, index, header, outputPath) {
```
[the remark this answers is not in the document]
Comment (yinonov) [reply apps/docs/transformers/code-block-demo.js:13]: 2nd that

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/transformers/code-block-demo.js (hunk 1 above)
Comment (yinonov) [inline apps/docs/transformers/code-block-demo.js:13]:
```suggestion
const generateCodeBlockDemo = function (pre, index, headEl, outputPath) {
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/transformers/code-block-demo.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    return document.documentElement.outerHTML;
+};
+
+const getHtml = (demoStr, codeStr, i, outputPath) => {
+    const codeBlockId = `${CBD_CODE_BLOCK}-${i}`;
+    const iframeSrc = getIframe(demoStr, codeBlockId, outputPath);
+
+    return `
+    <div class="${CBD_BASE}">
+        <iframe class="${CBD_DEMO}" src="${iframeSrc}" onload=onloadIframe(this)></iframe>
```
Comment (yinonov) [inline apps/docs/transformers/code-block-demo.js:44]:
```suggestion
        <iframe class="${CBD_DEMO}" src="${iframeSrc}" onload=onloadIframe(this) loading="lazy"></iframe>
```

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — COMMENTED** [review]
did you notice the network tasking? it has grown significantly

**Review by yinonov — APPROVED** [review]

**Pull request merged**